sdcm/sct_provision/instances_provider.py

```
@@ -56,9 +56,13 @@ def provision_instances_with_fallback(
             raise
 
     provisioned_instances = provisioner.get_or_create_instances(definitions=definitions)
-    for v_m in provisioned_instances:
+    for definition, v_m in zip(definitions, provisioned_instances):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)